src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
You should always provide a fallback font in case the main one doesn't load. According to the checklist, you should add a fallback font-family. It could be a system font.